test/algorithms/set_operations/difference/difference.cpp

```diff
@@ -636,7 +629,7 @@ int test_main(int, char* [])
 #if defined(BOOST_GEOMETRY_TEST_FAILURES)
     // Not yet fully tested for float and long double.
     // The difference algorithm can generate (additional) slivers
-    BoostGeometryWriteExpectedFailures(10, 7, 22, 8);
+    BoostGeometryWriteExpectedFailures(12, 5, 11, 6);
```

Comment:
(the 2 extra for rescaled is not a regression, one test now also tests symmetric difference which fails for rescaling but succeeds otherwise)

Reply:
sorry, I didn't get where the extra '2' comes from

Reply:
From symmetric difference which is now also tested for that case (resulting in 2 errors)